tests/pytests/test_issues.py

```diff
@@ -775,3 +775,24 @@ def test_mod5062(env):
   # verify using counter instead of sorter, even with explicit sort
   aggregate_profile = env.cmd('FT.PROFILE', 'idx', 'AGGREGATE', 'QUERY', 'hello', 'SORTBY', '1', '@t')
   env.assertEquals('Counter', aggregate_profile[1][4][2][1])
+
+def test_mod5252(env):
+  # Create an index and add a document
+  env.expect('FT.CREATE', 'idx', 'SCHEMA', 't', 'TEXT', 'n', 'NUMERIC').equal('OK')
```

Comment:
BTW, we allow to override `__key` with an alias such as 
`FT.CREATE idx2 SCHEMA t TEXT n NUMERIC AS __key`
And then it no longer returns the key, and as requested, returns the attribute - seems OK to me.

Reply:
I wondered if we should prevent this, or allow setting another key name, but I guess we can discuss this in the future